src/buildURL.js

```diff
@@ -19,6 +19,8 @@ const buildURL = {
         baseUrl += '&OPERATION-NAME=' + options.operationName;
         baseUrl += '&SERVICE-VERSION=1.0.0&RESPONSE-DATA-FORMAT=JSON';
         baseUrl += options.param ? '&' + options.param + '=' + options.name : '';
+        baseUrl += options.sortOrder ? '&sortOrder=' + options.sortOrder : '';
+        baseUrl += '&outputSelector(0)=SellerInfo';
```

Comment:
What this line does `&outputSelector(0)=SellerInfo` ?.  Can you point me to any documentation

Reply:
https://developer.ebay.com/DevZone/finding/CallRef/findItemsByKeywords.html
I added this line, because there wasn't the informations of the seller. So this provide me things that I wanted, like the seller's name ;)

Reply:
okay cool.. Thanks.